### PR TITLE
[BUG] Fix pyproject package structure and failures in tutorial notebooks

### PR DESCRIPTION
**Pull request by XinyuWuu**
fix https://github.com/jdb78/pytorch-forecasting/issues/1539

Removed `os.chdir("../../..")` in notebooks.

This PR should be merged after https://github.com/jdb78/pytorch-forecasting/pull/1599

### CODE REVIEW

**Comment by fkiraly** [comment]
@XinyuWuu, removal of chdir seems to break the imports - very strage, I would have expected them to work due to `pip install` including the package

**Comment by XinyuWuu** [comment]
@fkiraly The cause seems to be that `pytorch_forecasting` is installed as a module instead of a package. Strange.

And `build_tools` is recognized as package directory by setuptools by default so I have to exclude it.

And quite unintutive,

This wouldn't fine sub packages like `pytorch_forecasting.data`
```python
[tool.setuptools]
packages = ["pytorch_forecasting"]
````

File: docs/source/tutorials/stallion.ipynb [hunk 1]
```diff
@@ -1917,7 +1917,7 @@
    "source": [
     "# calcualte metric by which to display\n",
     "predictions = best_tft.predict(val_dataloader, return_y=True)\n",
-    "mean_losses = SMAPE(reduction=\"none\")(predictions.output, predictions.y).mean(1)\n",
+    "mean_losses = SMAPE(reduction=\"none\").loss(predictions.output, predictions.y[0]).mean(1)\n",
```
Comment (fkiraly) [inline docs/source/tutorials/stallion.ipynb:1920]: this is a workaround for the notebook, but not a fix for #1614, right?

File: docs/source/tutorials/ar.ipynb [hunk 2]
```diff
@@ -14,19 +14,25 @@
...
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/xinyu/DATA1_1T/anaconda3/envs/PF_310/lib/python3.10/site-packages/pytorch_forecasting/models/base_model.py:31: TqdmWarning: IProgress not found. Please update jupyter and ipywidgets. See https://ipywidgets.readthedocs.io/en/stable/user_install.html\n",
```
Comment (fkiraly) [inline docs/source/tutorials/ar.ipynb]: can we remove this warning?

File: docs/source/tutorials/stallion.ipynb (hunk 1 above)
Comment (XinyuWuu) [reply docs/source/tutorials/stallion.ipynb:1920]: It's a workaround.

File: docs/source/tutorials/ar.ipynb (hunk 2 above)
Comment (XinyuWuu) [reply docs/source/tutorials/ar.ipynb]: Removed

**Pull request merged**